src/Core/Silk.NET.Core/Native/Structs/ISequentialStream.cs

```diff
@@ -0,0 +1,128 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+using Silk.NET.Core.Native;
+using Silk.NET.Core.Attributes;
+using Silk.NET.Core.Contexts;
+using Silk.NET.Core.Loader;
+
+#pragma warning disable 1591
+
+namespace Silk.NET.Core.Native
+{
+    [Guid("0c733a30-2a1c-11ce-ade5-00aa0044773d")]
+    public unsafe partial struct ISequentialStream
```

Comment:
Is there no way we can add objidl.h to the headers for Core? These look like they were generated but then manually copied to core (judging by the lack of .gen.cs)

Reply:
We could add them to core, but more likely would be Win32Extras as they rely on FileTime. These were also written by hand, just following the pattern of a generated file.

Reply:
So just looking into it, first objidlbase.h is what we would be adding, objidl.h isn't generating the structures. The only issue I'm seeing is that if we include all of that we need some types from WTypesBase.h(_COAUTHINFO, which relies on _COAUTHIDENTITY, for a struct) and rpcdcep.h(_RPC_MESSAGE for a bunch of global functions). 
I think we can these two without needing any more, just feels like this is a lot to add for IStream. What do you think?

Reply:
I did add objidlbase.h to win32Extras with its dependancies, WTypesBase.h and rpcdcep.h. the rpcdcep.h types seem to not be read correctly by clang and only the define consts seem to be read correctly, but I did bring in the required structs in manually.